src/python/grpcio/grpc/extensions/__init__.py

```diff
@@ -0,0 +1,38 @@
+# Copyright 2015-2016 gRPC authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""gRPC Python extension API."""
+
+"""The client config of the gRPC extensions module.
+Its value is a proto defined by client_config.proto"""
+CHANNEL_ARG_GRPC_EXTENSIONS_CLIENT_CONFIG = "grpc.extensions.client_config"
+
+
+def secure_channel(target, credentials, options=None):
+    """Creates a secure Channel to a server.
+
+    Args:
+      target: The server address.
+      credentials: A ChannelCredentials instance.
+      options: An optional list of key-value pairs (channel args in gRPC Core
```

Comment:
channel args defined in grpc Core or the channel-management extensions

Reply:
It's and/or, as they can be combined.